src/software/proto/tbots_replay.proto

```diff
@@ -0,0 +1,7 @@
+syntax = 'proto2';
```

Comment:
nit: We generally are using proto3 unless otherwise required for our own message types

Reply:
fixed